### PR TITLE
bump core

### PR DESCRIPTION
**Pull request by mdulat**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4182,7 +4182,7 @@
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#d2bd049701544ed251c8f0a3fc21214ee5bbffae",
+      "version": "github:BrightspaceHypermediaComponents/activities#ef0871d8c50640fc82e96f61b11a43e605c2808d",
       "from": "github:BrightspaceHypermediaComponents/activities#semver:^3",
```
Comment (mdulat) [inline package-lock.json:4186]: @spandanabaruah

File: package-lock.json [hunk 2]
```diff
@@ -4385,14 +4384,16 @@
       }
     },
     "d2l-consistent-evaluation": {
-      "version": "github:BrightspaceHypermediaComponents/consistent-evaluation#aa84ee3b01564ac793bbd9ffaef4d51ef480df27",
+      "version": "github:BrightspaceHypermediaComponents/consistent-evaluation#9305610921dd78af739f3291a8253810feb83c8c",
       "from": "github:BrightspaceHypermediaComponents/consistent-evaluation#semver:^0",
```
Comment (mdulat) [inline package-lock.json:4388]: @rowansilver @dmackenz @AlexBedley

File: package-lock.json [hunk 3]
```diff
@@ -4441,7 +4508,7 @@
       }
     },
     "d2l-cpd": {
-      "version": "github:Brightspace/continuous-professional-development#5ea0f578be4766877ee3014167509e568d8f7597",
+      "version": "github:Brightspace/continuous-professional-development#dd897a10426d787b133d78593b5801a94cda2816",
       "from": "github:Brightspace/continuous-professional-development#semver:^1",
```
Comment (mdulat) [inline package-lock.json:4512]: @TomMichielsD2L @BraydonKains

File: package-lock.json [hunk 4]
```diff
@@ -4484,7 +4550,7 @@
       }
     },
     "d2l-datetime-picker": {
-      "version": "github:BrightspaceUI/datetime-picker#c23e458ae68b454445bbf44245d6c9f1a7923d07",
+      "version": "github:BrightspaceUI/datetime-picker#203d92f83a2f6a0651abd4baf0a1a63cdf2e05db",
       "from": "github:BrightspaceUI/datetime-picker#semver:^4",
```
Comment (mdulat) [inline package-lock.json:4554]: @AllanKerr

File: package-lock.json [hunk 5]
```diff
@@ -4599,7 +4665,7 @@
       }
     },
     "d2l-fetch-siren-entity-behavior": {
-      "version": "github:Brightspace/d2l-fetch-siren-entity-behavior#ce4d13543a216d631c6e946a1f33ec84a1a20c9c",
+      "version": "github:Brightspace/d2l-fetch-siren-entity-behavior#b25b04f61f7eeb98ccaae5b6efaeb0d7908caa4f",
       "from": "github:Brightspace/d2l-fetch-siren-entity-behavior#semver:^6",
```
Comment (mdulat) [inline package-lock.json:4669]: @omsmith

File: package.json [hunk 6]
```diff
@@ -11,8 +11,8 @@
     "build:babelHelpers": "babel-external-helpers > ./build/babel-helpers.js",
     "build:rename-define": "node ./cli/rename-define.js \"./build/**/*.js\"",
     "build:email-icons": "node ./cli/build-email-icons.js",
-    "build:wc": "cross-env NODE_OPTIONS=--max_old_space_size=4096 polymer build",
-    "build:wc-no-es5": "cross-env NODE_OPTIONS=--max_old_space_size=4096 polymer build --name \"es6-bundled\"",
+    "build:wc": "cross-env NODE_OPTIONS=--max_old_space_size=8192 polymer build",
+    "build:wc-no-es5": "cross-env NODE_OPTIONS=--max_old_space_size=8192 polymer build --name \"es6-bundled\"",
```
Comment (AllanKerr) [inline package.json:15]: Was this intentional? Looks like you might have committed part of your local config.

File: package-lock.json [hunk 7]
```diff
@@ -4843,7 +4909,7 @@
       "dev": true
     },
     "d2l-organizations": {
-      "version": "github:BrightspaceHypermediaComponents/organizations#fef15423ac2c561a5124674b47f5db84a33f98cf",
+      "version": "github:BrightspaceHypermediaComponents/organizations#bcbbc8dce6526cc164355b861c286ee17045a8a8",
       "from": "github:BrightspaceHypermediaComponents/organizations#semver:^5",
```
Comment (mdulat) [inline package-lock.json:4913]: @iFun

File: package-lock.json [hunk 8]
```diff
@@ -26823,7 +26899,7 @@
       "dev": true
     },
     "siren-sdk": {
-      "version": "github:BrightspaceHypermediaComponents/siren-sdk#635c8dc4baf0f6c23c50eec92417d3b8b43c2289",
+      "version": "github:BrightspaceHypermediaComponents/siren-sdk#ba482ee6078b41c876b9ce7d07e327d8167ba349",
       "from": "github:BrightspaceHypermediaComponents/siren-sdk#semver:^1",
```
Comment (mdulat) [inline package-lock.json:26903]: @pureooze @emil-furniss

File: package.json (hunk 6 above)
Comment (mdulat) [reply package.json:15]: It was intentional. Since I can't build with the old value I assumed others would have the same issue. Did I assume incorrectly?

Comment (AllanKerr) [reply package.json:15]: It seems to be build fine for me with `4096`. I thought `8192` might be a bit high for people running on machines with 16 gb of RAM.

File: package-lock.json (hunk 7 above)
Comment (iFun) [reply package-lock.json:4913]: thank you

File: package-lock.json [hunk 9]
```diff
@@ -4402,6 +4403,72 @@
           "resolved": "https://registry.npmjs.org/@adobe/lit-mobx/-/lit-mobx-0.0.2.tgz",
           "integrity": "sha512-OAA+Ji0ei5GBv6uNWPJCs1YN/WmCnOM0mPDFiB2iKLQPVi5/bbfcuPpzrJQR5n7lhFZiojh18dGGVd0pKPtEbg==",
           "dev": true
+        },
+        "d2l-html-editor": {
```
Comment (iFun) [inline package-lock.json:4407]: I think it failed because this part

File: package.json (hunk 6 above)
Comment (emil-furniss) [reply package.json:15]: fwiw, I've had to bump to `8192` when certain repos are linked locally (especially larger ones). BSI alone has been fine with `4096`.

File: package-lock.json (hunk 9 above)
Comment (mdulat) [reply package-lock.json:4407]: Right, I forgot to update Node, will re-do this in a sec...

File: package.json (hunk 6 above)
Comment (mdulat) [reply package.json:15]: Ah, ok thanks, will put it back to 4096.

File: package-lock.json (hunk 9 above)
Comment (svanherk) [reply package-lock.json:4407]: I believe this is related to problems in the `d2l-consistent-evaluation` repo itself.  The `package.json` there needs to be updated: https://github.com/Brightspace/brightspace-integration/pull/2013#discussion_r401677701
Oh nice, looks like Owen has a PR up: https://github.com/BrightspaceHypermediaComponents/consistent-evaluation/pull/17

Comment (svanherk) [reply package-lock.json:4407]: FYI @dmackenz @rowansilver

Comment (svanherk) [reply package-lock.json:4407]: And FYI @AlexBedley

File: package-lock.json (hunk 5 above)
Comment (omsmith) [reply package-lock.json:4669]: :+1:

**Pull request merged**